browser/ChangeLog.md

```diff
@@ -1,5 +1,21 @@
 Note: This is the change log file for Azure Storage JavaScript Client Library.
 
+2018.05 Version 2.9.100-preview
+
+ALL
+* Generated browser compatible JavaScript files based on Microsoft Azure Storage SDK for Node.js 2.9.0-preview.
+* Updated storage service version to 2017-11-09.
+* Added `/* eslint-disable */` to generated JS files to avoid eslint warnings when using with create-react-app.
+* Added `progress` event for `SpeedSummary` class, which will be triggered when every progress updates.
+
+BLOB
+* Add ```createBlobServiceWithBearerToken()``` to create ```BlobService``` object with bearer tokens such as OAuth access token.
```

Comment:
Seems changed to createBlobServiceWithTokenCredential(), and same for Queue?

Reply:
Good catch! Fixed in another commit